tests/zfs-tests/include/libtest.shlib

```diff
@@ -3203,29 +3204,28 @@ function get_objnum
 }
 
 #
-# Synchronize all the data in pool
+# Sync data to the pool
 #
 # $1 pool name
+# $2 boolean to force uberblock update, useful when there is no dirty data
+#    in the pool
 #
-function sync_pool #pool
+function sync_pool #pool <force>
 {
 	typeset pool=${1:-$TESTPOOL}
+	typeset force=false
```

Comment:
Let's simplify this by using the same style shorthand as in the line above.  Default this value to false unless $2 was provided in which case use that.

```
typeset force=${2:-false}
```